azure-pipelines.yml

```diff
@@ -57,5 +55,4 @@ jobs:
     env:
       npcap_oem_key: $(npcap_oem_key)
   - script: dotnet restore -s https://api.nuget.org/v3/index.json
-  # Disable WinDivert to avoid an internal CLR error due to secure boot having been enabled
```

Comment:
Changes to this files, do not look related to the PR

Reply:
@kayoub5 yep, but they are in a separate commit in any case. I can break it off into a separate pr if it helps with the review process.